docs/user/dashboard/lens.asciidoc

```diff
@@ -197,11 +199,15 @@ Annotations allow you to call out specific points in your visualizations that ar
 [role="screenshot"]
 image::images/lens_annotations_8.2.0.png[Lens annotations]
 
+Create the annotation. 
+
 . In the layer pane, click *Add layer > Annotations*.
 
 . Select the {data-source}.
 
-. To open the annotation options, click *Event*.
+. Click *Event*, or drag a field to the *Add an annotation* field in the *Annotations* layer pane.
```

Comment:
If this is an "or" situation, can we just give users the preferred method of doing this?